azerothcore/azerothcore-wotlk
# fix(DB/Quest): Rifle the Bodies

## CHANGES PROPOSED:
- Make 12440 require 12439
- Make 12000 NOT REQUIRE ANY QUESTS because all quests leading up should be breadcrumbs and not required.
- Make 12440, 11995, 12439 unavailable if you accept quest 12000 because they are BREADCRUMB QUESTS
- Make 11995 & 12440 exclude each other as they are the "same" quest

## TESTS PERFORMED:
- Builds
- Works in-game



## HOW TO TEST THE CHANGES:

1. .go c id 26673
2. See that you can pick up Rifle and Bodies
3. .go c id 27158
4. See that he offers A Disturbance In The West (Hand in the quest)
5. See that Rodney offers To Stars' Rest (Accept it)
6. .go c id 26440
7. See that she does not offer Your Presence is Required at Stars' Rest when you have To Stars' Rest
8. Abandon To Stars' Rest and see that she will offer a quest.
9. .q add 12000
10. See that she no longer offers a quest. (Only needed to be tested here since it uses the same logic on all places)


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
